### ISSUE TITLE
nanopub.org not working

### ISSUE THREAD

**Issue by nicola883** [fair-workflows/nanopub#161]
Hi.
It seems that the website nanopub.org is not reachable:
http://nanopub.org/
I think that it is an essential resource for nanopublications.
Is it still being maintained? 
Is there someone that can fix it?
Thank you!

**Comment by raar1** [comment]
Hi @nicola883! Thanks for reporting this. That site is indeed still being maintained, and my understanding is that @tkuhn is looking into this now. That site was mostly just information about nanopublications and how they are used. You can see the contents on a temporary version at <https://nanopub.net/> , but hopefully the nanopub.org links start working again soon.

In any case, the actual nanopub servers (i.e. that store/retrieve nanopubs) appear to still be up (see e.g. <http://server.nanopubs.lod.labs.vu.nl/> )

**Comment by tkuhn** [comment]
Yes, it's a known issue. I have been trying to figure out who exactly owns the domain, which has been more difficult to find out than expected. I hope we get to the bottom of this soon, and in the meantime, nanopub.net has all the info (and more) that the nanopub.org had before it went down.

**Comment by dennisvang** [comment]
Unfortunately, it appears that https://nanopub.org is still down (or again, perhaps).
Is nanopub.*net* the new nanopub.*org*?

**Comment by tkuhn** [comment]
Yes, nanopub.net is the new nanopub.org. At least until the person who own nanopub.org resurfaces...

**Comment by dennisvang** [comment]
@tkuhn Thanks for the confirmation.

Is there a reason the documentation links have not been updated accordingly?

**Comment by tkuhn** [comment]
Which documentation links are you refering to?

**Comment by dennisvang** [comment]
@tkuhn Initially I was thinking of the ones below:

https://github.com/fair-workflows/nanopub/blob/2072905fe79b779d04e799c29344a54cc14b8d9b/docs/getting-started/what-are-nanopubs.md?plain=1#L7-L10

However, there's more, see e.g. [a quick search for `nanopub.org`][1]

For example, I wonder, isn't it problematic if you don't control the [namespace][2] domain?

[1]: https://github.com/search?q=repo%3Afair-workflows%2Fnanopub%20nanopub.org&type=code
[2]: https://github.com/fair-workflows/nanopub/blob/2072905fe79b779d04e799c29344a54cc14b8d9b/nanopub/namespaces.py#L6

**Comment by ashleycaselli** [comment]
Closed by 9bee7ee2b96fcb17741af90b63d37c66eea662cc
